src/develop/imageop.c

```diff
@@ -117,6 +117,11 @@ static char *default_description(struct dt_iop_module_t *self)
   return g_strdup("");
 }
 
+static const char *default_aliases(void)
+{
+  return g_strdup("");
```

Comment:
shouldn't this return just `""` not `g_strdup("")`? `g_strdup` without `g_free` causes memleak...

Reply:
`""` without `g_strdup` causes segfault…

Reply:
Huh... OK... No problem then.

Reply:
I'll have a look, you can't have a const and g_strdup() very suspicious !

Reply:
The only place where module->aliases() is called already to a g_strdump:

`        g_hash_table_insert(module_aliases, module->op, g_strdup(module->aliases()));`

So I bet you have fixed the issue in two places maybe....

Reply:
All is well with the g_strdup() in default_aliases() so I have removed it.